primitives/core/src/crypto.rs

```diff
@@ -502,7 +512,7 @@ impl ByteArray for AccountId32 {
 	const LEN: usize = 32;
 }
 
-#[cfg(feature = "std")]
+#[cfg(all(feature = "serde", feature = "full_crypto"))]
```

Comment:
Why do we need this with `serde`?

Reply:
Because of the `to_ss58check()` function within serialzation